bracket-pathfinding/src/dijkstra.rs

```diff
@@ -269,60 +269,6 @@ impl DijkstraMap {
         }
     }
 
-    #[cfg(feature = "threaded")]
```

Comment:
This was a duplicated function causing issues when setting `threaded` feature on pathfinding